recipe/passwordless/passwordlessfunctions.go

```diff
@@ -103,7 +104,18 @@ func DefaultCreateAndSendCustomTextMessage(appInfo supertokens.NormalisedAppinfo
 			return err
 		}
 		if resp.StatusCode == 429 {
-			return errors.New("Free daily SMS quota reached. If using our managed service, please create a production environment to get dedicated API keys for SMS sending, or define your own method for sending SMS.")
+			smsData, err := json.Marshal(data["smsInput"])
+			if err != nil {
+				return err
+			}
+			fmt.Println("Free daily SMS quota reached. If using our managed service, please create a production environment to get dedicated API keys for SMS sending, or define your own method for sending SMS. For now, we are logging it below:")
+			fmt.Printf("SMS content: %s\n", string(smsData))
```

Comment:
Can you show a screenshot of how this looks like?

Reply:
<img width="870" alt="image" src="https://user-images.githubusercontent.com/650429/170661343-89a102e9-cf32-4ed6-97c4-4d7869f7d7f0.png">

Reply:
You need to add a new line between the two messages.